Build/run.sh

```diff
@@ -48,12 +55,25 @@ while [[ $# -gt 0 ]]; do
         ;;
     esac
 done
+if [ "$BRANCH" == "1" ];
+then
+	CONTNAME="$(whoami)-rzg2l_vlp_v3.0.0_$(git branch --show-current)"
+else
+	CONTNAME="$(whoami)-rzg2l_vlp_v3.0.0"
+fi
 #Create OUTDIR if iot doesn't exist
+#echo "VERBOSE: ${VERBOSE} NO: ${NO}"
+#exit
```

Comment:
Do we need these?

Reply:
Good point! No we obviously don't! I'll address accordingly!